pkg/storage/secret/metadata/outbox_store.go

```diff
@@ -125,21 +140,25 @@ func (s *outboxStore) ReceiveN(ctx context.Context, n uint) ([]contracts.OutboxM
 	defer func() { _ = rows.Close() }()
 
 	messages := make([]contracts.OutboxMessage, 0)
+	messageIDs := make([]string, 0, len(messages))
```

Comment:
This is not being used as well right? 🤔

Reply:
Removed. I was trying something before 😆